django/db/models/fields/files.py

```diff
@@ -268,6 +269,20 @@ def _check_primary_key(self):
         else:
             return []
 
+    def _check_upload_to(self):
+        # Added system check based on #27358
+        if self.upload_to and self.upload_to[0] == '/':
```

Comment:
might want to add a `and isinstance(self.upload_to, six.string_types)` as `upload_to` can be a callable.